src/utils/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const gamesApi = axios.create({
+  baseURL: "https://nc-games-w79g.onrender.com/api",
+});
+
+export const getReviews = async () => {
+  const { data } = await gamesApi.get("/reviews");
```

Comment:
Great to see you using async/await here and destructure straight off the response.